python/sglang/srt/layers/layernorm.py

```diff
@@ -139,7 +140,7 @@ def extra_repr(self):
         return f"{tuple(self.weight.shape)}, eps={self.eps}"
 
 
-if not _is_cuda:
+if not _is_cuda and not _is_hpu:
```

Comment:
same as the other comments, maybe it will be better to move _is_cuda and _is_hpu into "_is_support_sgl_kernel" and use them here?

Reply:
same as above response